docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -64,6 +64,52 @@ The dashboard is no longer accessible, even with the link, until you make it sha
 
 The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
 
+## Email Sharing
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the url.
+
+### Enable email sharing
+
+> **Note:** For Grafana Cloud, you will need to contact support to have the feature enabled.
+
+Add the `publicDashboardsEmailSharing` feature toggle to your `custom.ini` file.
+
+```
+[feature_toggles]
+publicDashboardsEmailSharing = true
+```
+
+### Inviting a viewer
+
+1. Click **Only specified people**.
+2. Enter the email you want to share the public dashboard with.
+3. Click **Invite**.
+4. They will receive an email with a one-time use link. This link must be used within **1 hour** or it will be considered expired. Once the link is used, it will give the viewer access to the public dashboard for **30 days**.
+
+### Viewers requesting access
+
+If a viewer without access tries to navigate to the public dashboard, they will be asked to request access by providing their email. They will receive an email with a new one-time use link if the email they provided has already been invited to view the public dashboard and has not been revoked.
+
+### Revoking access for a viewer
+
+1. Click **revoke** on the viewer you'd like to revoke access for.
+2. This will immediately prevent the user from accessing the public dashboard or using any existing one-time use links they may have.
+
+### Resharing access for a viewer
```

Comment:
```suggestion
### Renew access for a viewer
```

Reply:
I'm curious about why change `Reshare` by `Renew`, we don't use it in other places. I think `Renew` could imply something different.

Reply:
I was trying to avoid some awkward wording, but if "renew" could imply something different, then we should change it. Perhaps, since this is the word we use for first giving a person access:
```suggestion
### Reinvite a viewer
```